sdk/js-cloud-server/README.md

```diff
@@ -0,0 +1,19 @@
+# DevCycle Javascript Cloud Bucketing Server SDK
+
+This SDK is used to integrate DevCycle with your Javascript server-side application, where a NodeJS runtime isn't available.
+Example use-cases would be Cloudflare Workers and other edge-worker runtimes.
+
+This SDK makes API requests to DevCycle's [Bucketing API](https://docs.devcycle.com/bucketing-api/) to perform bucketing.
+See our [System Architecture Documentation](https://docs.devcycle.com/introduction/architecture#cloud-bucketing-server-sdk-architecture) for more details.
+
+### Installation
+
+Our library can be found on npm and installed by the following:
+
+```
+npm install @devcycle/js-cloud-server-sdk"
+```
+
+### Usage
+
+To find usage documentation, visit our [docs](https://docs.devcycle.com/docs/sdk/server-side-sdks/node#usage).
```

Comment:
I'm actually thinking we may not create its own page on our docs. As its essentially going to a part of the NodeJS SDK and the Edge Worker SDK, I think it would be confusing to have to also explain what this is and when to use it. Just need to make sure we explain that well in this README.